### PR TITLE
Release 0.7.0

### PR DESCRIPTION
**Pull request by JeanTracker**
NuguSDK release 0.7.0

Signed-off-by: JeanTracker <hyojoong.kim.jean@gmail.com>

### CODE REVIEW

File: packaging/bionic/changelog [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+  * Upgraded ASR interface to v1.1
+  * Upgraded AudioPlayer interface to v1.2
+  * Upgraded Text interface to v1.1
+  * Upgraded curl to v7.69.1
+  * Removed Display agent
+  * Removed Location agent
+  * Removed Delegation agent
+  * Removed Extension agent
+
+ -- nugulinux <nugulinux@gmail.com>  Mon, 31 Mar 2020 14:30:00 +0900
```
Comment (kimhyungrok) [inline packaging/bionic/changelog]: Please check Day of the week.

File: packaging/xenial/changelog [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+  * Upgraded ASR interface to v1.1
+  * Upgraded AudioPlayer interface to v1.2
+  * Upgraded Text interface to v1.1
+  * Upgraded curl to v7.69.1
+  * Removed Display agent
+  * Removed Location agent
+  * Removed Delegation agent
+  * Removed Extension agent
+
+ -- nugulinux <nugulinux@gmail.com>  Mon, 31 Mar 2020 14:30:00 +0900
```
Comment (kimhyungrok) [inline packaging/xenial/changelog]: Please check Day of the week.

File: packaging/bionic/changelog (hunk 1 above)
Comment (JeanTracker) [reply packaging/bionic/changelog]: Thanks.

File: packaging/xenial/changelog (hunk 2 above)
Comment (JeanTracker) [reply packaging/xenial/changelog]: Thanks.

**Pull request merged**